lib/src/base-styles/_nx-page-header.scss

```diff
@@ -39,61 +39,44 @@ $main-header-product-version-row-height: 1fr;
 .nx-product {
   align-self: center;
   box-sizing: border-box;
-  color: $nx-off-white;
-  display: -ms-grid;
-  display: grid;
-  grid-column-gap: $nx-spacing-xxs;
-  -ms-grid-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
-  grid-template-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
-  -ms-grid-columns: $main-header-product-height $nx-spacing-xxs auto;
-  grid-template-columns: $main-header-product-height auto;
-  grid-template-areas:
-    "logo title"
-    "logo version";
+  color: $main-header-color;
+  display: flex;
   height: $main-header-product-height;
   text-decoration: none;
   min-width: $nx-sidebar-width;
 }
 
 .nx-product__logo {
-  -ms-grid-row: 1;
-  -ms-grid-row-span: 2;
-  -ms-grid-column: 1;
-  -ms-grid-row-align: center;
   align-self: center;
   box-sizing: border-box;
-  grid-area: logo;
-  height: 100%;
-  width: 100%;
+  height: 20px;
+  margin-top: $nx-spacing-xxs;
+  width: 20px;
 }
 
-.nx-product__name {
-  -ms-grid-row: 1;
-  -ms-grid-column: 3;
+.nx-product__sonatype-wordmark {
   @include semi-bold();
+  align-self: center;
+  font-size: 22px;
+  margin-left: $nx-spacing-xxs;
+}
 
-  -ms-grid-row-align: center;
+.nx-product__vertical-divider {
+  align-self: center;
+  font-size: 21px;
+  margin-left: $nx-spacing-sm;
+}
 
+.nx-product__name {
   align-self: center;
-  font-size: 23px;
-  grid-area: title;
-  -ms-grid-column-align: left;
-  justify-self: left;
-  line-height: normal;
-  text-transform: lowercase;
+  font-size: 26px;
+  margin-left: $nx-spacing-sm;
   white-space: nowrap;
 }
 
 .nx-product__version {
-  -ms-grid-row: 2;
-  -ms-grid-column: 3;
-  @include bold();
-
-  -ms-grid-row-align: start;
-  align-self: start;
+  align-self: center;
   font-size: 11px;
```

Comment:
12 in zeplin

Reply:
> 12 in zeplin

Fixed: https://github.com/sonatype/sonatype-react-shared-components/pull/30/commits/3549fe76834f168a1be07fb99bd3835ef3ae6c57